query-model/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/querymodel/jpql/expression/impl/JpqlModulo.kt

```diff
@@ -0,0 +1,13 @@
+package com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl
+
+import com.linecorp.kotlinjdsl.Internal
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+
+/**
+ * Expression that calculates the mod of a integer [value1] to a specified integer [value2].
+ */
+@Internal
+data class JpqlModulo<T : Int> internal constructor(
```

Comment:
Modulo seems more appropriate, but the JPQL specification doesn't mention modulo, so we should rename the class JpqlMod.

Reply:
Reflected now.